docs/test.md

```diff
@@ -0,0 +1,88 @@
+<h1>Testing</h1>
```

Comment:
I prefer to use markdown and dismiss the use of HTML tags

Reply:
This is a technique to reduce the ToC in Readthedocs - I'm only using it to make the documentation easier to read